src/__tests__/Mutation-test.js

```diff
@@ -48,6 +48,16 @@ describe('Mutation Generators', function() {
     expect(result instanceof Parse.Promise).toBe(true);
   });
 
+  it("fails to generate a Set Mutation with multiple GeoPoints", function () {
+      expect(function () {
+        var id = new Id('Klass', '01');
+        Mutation.Set(id, {
+          g1: new Parse.GeoPoint(),
+          g2: new Parse.GeoPoint(),
+        });
+      }).toThrow();
```

Comment:
When testing that an exception is thrown, I find it safest to test for the error string as well.
So, `.toThrow()` becomes `.toThrow('There can only be 1 GeoPoint when mutating an object.')`
Otherwise, a different bug can go unnoticed as long as something breaks somewhere in the call.